server/lobbyconnection.py

```diff
@@ -715,8 +747,17 @@ def _set_user_agent_and_version(self, user_agent, version):
 
             async with self._db.acquire() as conn:
                 result = await conn.execute(
-                    "SELECT url, tooltip FROM `avatars` "
-                    "LEFT JOIN `avatars_list` ON `idAvatar` = `avatars_list`.`id` WHERE `idUser` = %s", (self.player.id,))
+                    select([
+                        avatars_list.c.url,
+                        avatars_list.c.tooltip
+                    ]).select_from(
+                        avatars.outerjoin(
+                            avatars_list
+                        )
+                    ).where(
+                        avatars.c.idUser == self.player.id
+                    )
+                )
```

Comment:
Imho, you made different query here. Please check what print(str(query)) does.

Imho select_from will generate whole table like select something from cartesian product, and then filter it.

Reply:
You are wrong. https://docs.sqlalchemy.org/en/13/core/tutorial.html#using-joins

Reply:
Here's `print(query)`:
```
SELECT avatars_list.url, avatars_list.tooltip 
FROM avatars LEFT OUTER JOIN avatars_list ON avatars_list.id = avatars."idAvatar" 
WHERE avatars."idUser" = :idUser_1
```